docs/release-notes/7.17.asciidoc

```diff
@@ -2,14 +2,24 @@
 == 7.17
 
 [discrete]
-[[release-notes-7.17.1]]
-=== 7.17.1
+[[release-notes-7.17.2]]
+=== 7.17.2
 
 [discrete]
-[[bug-fixes-7.17.1]]
+[[bug-fixes-7.17.2]]
 ==== Bug fixes and enhancements
+* Fixes an {endpoint-sec} integration bug that prevented benign Windows files from being deleted under certain circumstances.
+* The primary signer for `elastic-endpoint.exe` has changed from ELASTICSEARCH B.V. to Elasticsearch, Inc.
```

Comment:
Can this include a link to this issue? https://github.com/elastic/endpoint/issues/15